protocol/src/main/resources/cluster-management-protocol.xml

```diff
@@ -59,4 +60,20 @@
     <data name="brokerVersion" id="11" type="varDataEncoding"/>
   </sbe:message>
 
+  <sbe:message name="BackupListResponse" id="5">
+    <!-- This response contains a subset of fields from BackupStatusResponse  -->
+    <group name="backups" id="1" dimensionType="groupSizeEncoding">
+      <field name="backupId" id="1" type="int64"/>
+      <field name="status" id="2" type="BackupStatusCode"/>
+      <field name="partitionId" id="3" type="uint16"/>
+      <!-- failure reason will have a valid value if status == FAILED -->
+      <data name="failureReason" id="8" type="varDataEncoding"/>
```

Comment:
:wrench: Consider adding an offset here in case we decide to add more fields

Reply:
What do you mean?

Reply:
Huh, I missed that there already is a small offset. Previous field id is 3, this one is 8.

I think I was wrong anyway, because if we add new fields later, they can have arbitrary ids, they don't have to be ordered. So please ignore my comment :sweat_smile:

Reply:
Ok :) id 8 was not intentional anyway. It was just copy-paste from BackupStatusResponse. I will leave it as it is.